example_iterator.py

```diff
@@ -2,42 +2,43 @@
 
 import settings
 from src.filenames.file_name_generator import OutputFileNameGenerator
-from src.loaders.normalize_scaler_loader import NormalizeScalerLoader
 from src.tif_model_iterator import tif_kernel_iterator
 import glob
 
 if __name__ == "__main__":
     filename = settings.MODEL_PATH
-    tif_file = settings.TIF_FILE
+    tif_file_regex = settings.TIF_FILE_INPUT_REGEX
     output_path = settings.OUTPUT_PATH
-    output_file_name = settings.OUTPUT_FILENAME
-
-    artefacts = pickle.load(open(filename, "rb"))
-    if isinstance(artefacts, dict):
-        loaded_model = artefacts["model"]
-        loaded_scaler = artefacts["scaler"]
-    else:
-        loaded_model = artefacts
-        path_to_scaler = settings.PATH_TO_SCALER
-        column_names = settings.COLUMN_NAMES
-
-        scaler_loader = NormalizeScalerLoader(
-            path_to_scaler_files=path_to_scaler,
-            tif_filepath=tif_file,
-            column_names=column_names,
-        )
-        loaded_scaler = scaler_loader.load()
 
-    output_file_name_generator = OutputFileNameGenerator(
-        output_path=output_path, output_file_name=output_file_name
-    )
+    print(filename)
+    loaded_model = pickle.load(open(filename, "rb"))
+
+    tif_files = [file for file in glob.glob(tif_file_regex)]
+
+    for tif_file in tif_files:
+        print("----------")
+        print(tif_file)
+
+        tif_file = tif_file.replace("\\", "/")
```

Comment:
Suggestion: Why would you not add this logic to the OutputFileNameGenerator instead? That's what it's for.

Reply:
this the problem with glob, it seems to sometimes return windows paths

Reply:
I meant, all the logic here line 22 to 27. That seems like it should be in the output_filename_generator.

Reply:
i agree

Reply:
i still think the user should have control about what the output of the file extension should be.
New commit which is a bit more cleanup

Reply:
This is not exactly what I meant, I meant changing the actual OutputFileNameGenerator class.